pkg/task/task.go

```diff
@@ -791,19 +794,42 @@ func (m *taskExecutor) deleteExtnV1B1Deployment() (err error) {
 	return
 }
 
+// getExtnV1B1ReplicaSet will get the Replicaset as specified in the RunTask
+func (m *taskExecutor) getExtnV1B1ReplicaSet() (err error) {
+	client := replicaset.KubeClient(
+		replicaset.WithNamespace(m.getTaskRunNamespace()))
+
+	rs, err := client.GetRaw(m.getTaskObjectName())
```

Comment:
can we combine both the above with proper indentation:
```go
rs, err := replicaset.
  KubeClient(replicaset.WithNamespace(m.getTaskRunNamespace())).
  GetRaw(m.getTaskObjectName())
```

Reply:
done